File: 04_sorting/04_01_selection_sort/selection_sort.py
def selection_sort(arr: list):
    for i in range(len(arr) - 1):
        for j in range(i + 1, len(arr)):
            if arr[j] < arr[i]:
                arr[i], arr[j] = arr[j], arr[i]
                
elements = [13, 46, 24, 52, 20, 9]
print("Before:-")
print(elements)
selection_sort(elements)
print("After:-")
print(elements)

File: 04_sorting/04_01_selection_sort/selection_sort.js
const selection_sort = (arr = []) => {
    for (let index = 0; index < arr.length; index++) {
        for (let j_index = 0; j_index < arr.length; j_index++) {
            if (arr[j_index] > arr[index]) {
                [arr[j_index], arr[index]] = [arr[index], arr[j_index]]
            }
        }
    }
}

const array = [13, 46, 24, 52, 20, 9]

console.log("Before:-");
console.log(array);
selection_sort(array)
console.log("After:-");
console.log(array);